common/src/View/MousePreferencePane.cpp

```diff
@@ -88,7 +89,7 @@ namespace TrenchBroom {
             m_downKeyEditor = new KeySequenceEdit(1);
             m_downKeyEditor->setSizePolicy(QSizePolicy::Fixed, QSizePolicy::Preferred);
 
-            m_flyMoveSpeedSlider = new SliderWithLabel(256, 512);
+            m_flyMoveSpeedSlider = new SliderWithLabel(0, 512);
```

Comment:
Why don't we use the same units as the preference does internally, i.e. 0 to 2, rather than scale it to 0 to 512?

Reply:
Because that will give us too few ticks. I have set it to 0...100 now, though.